SMLHelper/MonoBehaviours/CustomCreature.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace SMLHelper.V2.MonoBehaviours
+{
+    using UnityEngine;
+
+    public class CustomCreature : MonoBehaviour
+    {
+        public float scale;
+
+        void Update()
+        {
+	     // At the moment all this does is fix the scale, it will likely have more functionality later
+            transform.localScale = new Vector3(scale, scale, scale);
```

Comment:
Is this this necessary to run on every update call?  
Is there any performance penalty to doing so?

Reply:
I have tested it in Start() and Awake(), and it leads to unexpected results. Namely some fish are the correct scale while others are still unscaled. I will look into doing it dynamically in Update()